src/codec_svt.c

```diff
@@ -175,7 +188,11 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
 
         // In order for SVT-AV1 to force keyframes by setting pic_type to
         // EB_AV1_KEY_PICTURE on any frame, force_key_frames has to be set.
+#if SVT_AV1_CHECK_VERSION(3, 0, 0)
+        svt_config->force_key_frames = true;
+#else
         svt_config->force_key_frames = TRUE;
+#endif
```

Comment:
I believe in the case of [c99, section 7.16][1], `true` should always expand to `1`, and we always used `1` for `TRUE`, so you should be able to unconditionally just use `true` with no problem regardless of SVT-AV1's version.

[1]: https://www.open-std.org/jtc1/sc22/WG14/www/docs/n1256.pdf

Reply:
done